src/LandingPage.css

```diff
@@ -28,16 +28,84 @@ h1, h3 {
     padding-left: 35px;
 }
 
+.round-input{
+    background-color:rgb(196, 197, 201);
+    border:2px solid;
+    color: #6D6E71;
+    padding: 20px;
+    text-align:center;
+    text-decoration:none;
+    display:inline;
+    font-size: 16px;
+    margin: 4px 2px;
+    height:60px;
+    border-radius: 8px;
+}
+
+.centered{
```

Comment:
This can become margin: 0 auto

Reply:
I've removed this, bootstrap css has made the .centered class obsolete